tests/acceptance/common.py

```diff
@@ -36,7 +36,7 @@
 configuration = {}
 
 
-def _start_qemu(qenv=None, conn=None):
+def _start_qemu(qenv=None, conn=None, qemu_wrapper=None):
```

Comment:
could we have at least a bit of comment here on what qemu_wrapper is?

Reply:
Okey.